src/org/ow2/proactive_grid_cloud_portal/scheduler/client/TestPortal.java

```diff
@@ -0,0 +1,111 @@
+ */
+package org.ow2.proactive_grid_cloud_portal.scheduler.client;
+
+import org.ow2.proactive_grid_cloud_portal.common.client.JSUtil;
+import org.ow2.proactive_grid_cloud_portal.common.client.Settings;
+import org.ow2.proactive_grid_cloud_portal.scheduler.shared.SchedulerConfig;
+
+import java.util.Map;
+
+import com.google.gwt.core.client.EntryPoint;
+import com.google.gwt.core.client.GWT;
+import com.google.gwt.event.dom.client.ClickEvent;
+import com.google.gwt.event.dom.client.ClickHandler;
+import com.google.gwt.event.dom.client.DoubleClickEvent;
+import com.google.gwt.event.dom.client.DoubleClickHandler;
+import com.google.gwt.event.dom.client.DragEnterEvent;
+import com.google.gwt.event.dom.client.DragEnterHandler;
+import com.google.gwt.event.dom.client.DragOverEvent;
+import com.google.gwt.event.dom.client.DragOverHandler;
+import com.google.gwt.user.client.Event;
+import com.google.gwt.user.client.rpc.AsyncCallback;
+import com.google.gwt.user.client.ui.Grid;
+import com.google.gwt.user.client.ui.Panel;
+import com.google.gwt.user.client.ui.RootLayoutPanel;
+import com.google.gwt.user.client.ui.SimplePanel;
+import com.google.gwt.user.client.ui.VerticalPanel;
+import com.google.gwt.user.client.ui.Widget;
+import com.smartgwt.client.util.SC;
+import com.smartgwt.client.widgets.Canvas;
+import com.smartgwt.client.widgets.layout.VLayout;
+import com.smartgwt.client.widgets.toolbar.ToolStrip;
+import com.smartgwt.client.widgets.toolbar.ToolStripButton;
+
+
+public class TestPortal implements EntryPoint {
+
+    public void onModuleLoad() {
+
+        VLayout rootLayout = new VLayout();
+        ToolStrip toolStrip = new ToolStrip();
+        toolStrip.setWidth(600);
+        toolStrip.setHeight(600);
+        ToolStripButton button = new ToolStripButton("Hello very long text to fill space");
+//        button.addBitlessDomHandler(new DragEnterHandler() {
+//            @Override
+//            public void onDragEnter(DragEnterEvent event) {
+//                System.out.println("drag eenter" + event);
+//            }
+//        }, DragEnterEvent.getType());
+        toolStrip.addButton(button);
+
+        Canvas panel = new Canvas();
+        panel.setWidth("700px");
+        panel.setHeight("200px");
+//        panel.getElement().getStyle().setBackgroundColor("#0000FF");
+
+        rootLayout.addMember(toolStrip);
+        rootLayout.addMember(panel);
+        rootLayout.draw();
+//        Event.addNativePreviewHandler(new Event.NativePreviewHandler() {
+//            @Override
+//            public void onPreviewNativeEvent(Event.NativePreviewEvent event) {
+//                System.out.println(event.getNativeEvent().getType());
+//            }
+//        });
+        button.addBitlessDomHandler(new DragOverHandler() {
+            @Override
+            public void onDragOver(DragOverEvent event) {
+                System.out.println("PANEL DRAG" + event);
+            }
+        }, DragOverEvent.getType());
+
```

Comment:
There is no assert in this test, it doesn t look like is actually testing the portal